### PR TITLE
Send Slack notifications on deploys

### PR DESCRIPTION
**Pull request by ekmartin**

### CODE REVIEW

File: src/server/deploy.bs [hunk 1]
```diff
@@ -3,8 +3,8 @@ import bluebird as Promise
 
 child_process = Promise.promisifyAll(child)
 
-python = '#{__dirname}/venv/bin/python'
-deployScript = '#{__dirname}/deploy.py'
+python = '#{process.cwd()}/venv/bin/python'
```
Comment (ekmartin) [inline src/server/deploy.bs]: This needs https://github.com/webkom/puppet-abakus/pull/76 to work.

File: src/server/notify.bs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+notifySuccess = (projectName, source) ->
+  text = 'Successfully deployed #{projectName} (source: #{source})'
+  notifySlack('success', text)
+
+notifyError = (projectName, source, error) ->
+  text = 'Failed deploying #{projectName} (source: #{source})'
+  notifySlack('alert', text, error)
+
+export {
```
Comment (relekang) [inline src/server/notify.bs]: Please put the export statement right below the import statements.

File: src/server/notify.bs [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import slack-notify as slackNotify
+
+notifySlack = (type, text, error) ->
+  if process.env.SLACK_URL
+    slackGateway = slackNotify(process.env.SLACK_URL)
+
+    options = {
+      text: text,
```
Comment (relekang) [inline src/server/notify.bs]: Please remove the `,` :)

**Comment by ekmartin** [comment]
Moved all the imports to the top now, also replaced our local check-env with the npm published one.

File: projects.json [hunk 4]
```diff
@@ -0,0 +1,47 @@
+{
```
Comment (relekang) [inline projects.json]: Did you mean to push this?

**Comment by relekang** [comment]
:clap: :cake:

**Pull request merged**